lib/espresso-runner.js

```diff
@@ -117,7 +120,37 @@ class EspressoRunner {
     logger.info(`Repackaged espresso server ready: '${this.modServerPath}'`);
   }
 
+  async cleanupSessionLeftovers () {
+    logger.debug('Performing cleanup of automation leftovers');
+
+    try {
+      const {value} = await request.get({
+        url: `http://${this.host}:${this.systemPort}/wd/hub/sessions`,
+        timeout: 500,
+        json: true,
+      });
+      const activeSessionIds = value.map((sess) => sess.id);
+      if (activeSessionIds.length) {
+        logger.debug(`The following obsolete sessions are still running: ${JSON.stringify(activeSessionIds)}`);
+        logger.debug('Cleaning up the obsolete sessions');
+        await B.all(activeSessionIds.map((id) =>
+          request.delete({
+            url: `http://${this.host}:${this.systemPort}/wd/hub/session/${id}`,
```

Comment:
`http://${this.host}:${this.systemPort}/session/${id}`?

Reply:
fixed